docs/en/operation/Troubleshooting.md

```diff
@@ -241,6 +241,14 @@ The parsable datetime formats include below:
 1. `--start-time <datetime>` specifies a datetime before with the log files can be ignored.
 A log file will be ignored if the last modified time is before this start time.
 
+
+## Alluxio limitation on file names
```

Comment:
is this limitation for file names or the full file path (ex. directory names)?

Reply:
for the full path